tests/foreman/ui_airgun/test_oscappolicy.py

```diff
@@ -117,3 +118,77 @@ def test_positive_check_dashboard(session, module_host_group, module_loc,
             'HostBreakdownChart']['hosts_breakdown'].split(" ", 1)
         assert host_breakdown_chart[0] == '100%'
         assert host_breakdown_chart[1] == 'Not audited'
+
+
+@tier2
+@upgrade
+def test_positive_end_to_end(
+        session, module_host_group, module_loc, module_org,
+        oscap_content_path, oscap_tailoring_path):
+    """Perform end to end testing for oscap policy component
+
+    :id: 39c26f89-3147-4f27-bf5e-810f0ba721d8
+
+    :expectedresults: All expected CRUD actions finished successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    name = '{} {}'.format(gen_string('alpha'), gen_string('alpha'))
+    new_name = gen_string('alpha')
+    description = gen_string('alpha')
+    oscap_content_title = gen_string('alpha')
+    tailoring_name = gen_string('alpha')
+    profile_type = 'Red Hat Corporate Profile for Certified Cloud Providers (RH CCP)'
+    tailoring_type = 'Common Profile for General-Purpose Systems [CUSTOMIZED1]'
```

Comment:
The tailoring file is different profile then the profile used. Although , this is a workflow issue and test would still pass. 
Such issues would be addressed by https://github.com/SatelliteQE/robottelo/issues/6849